inspirehep/modules/workflows/workflows/article.py

```diff
@@ -511,9 +500,9 @@
 
         # stop the matched wf and continue this one
         [
-            stop_matched_holdingpen_wfs,
-            mark('stopped-matched-holdingpen-wf', True),
-            save_workflow
+            mark('stopped-matched-holdingpen-wf', False),
+            save_workflow,
+            stop_processing
```

Comment:
I'm confused, why did you change this one?

Reply:
cause we talked to finish submission workflows, at least it's what I understood.